extensions/observability-devservices/testlibs/devresource-lgtm/src/main/java/io/quarkus/observability/devresource/lgtm/LgtmResource.java

```diff
@@ -53,23 +92,13 @@ protected LgtmContainer defaultContainer() {
     @Override
     public Map<String, String> doStart() {
         String host = container.getHost();
-        int otlpPort = container.getOtlpPort();
-
-        //Set non Quarkus properties for convenience and testing.
         Map<String, String> containerConfigs = new HashMap<>();
-        containerConfigs.put("grafana.endpoint", String.format("http://%s:%s", host, container.getGrafanaPort()));
-        containerConfigs.put("otel-collector.url", String.format("%s:%s", host, otlpPort));
 
-        // set relevant properties for Quarkus extensions directly
-        if (catalog != null && catalog.hasOpenTelemetry()) {
-            containerConfigs.put("quarkus.otel.exporter.otlp.endpoint", String.format("http://%s:%s", host, otlpPort));
-            containerConfigs.put("quarkus.otel.exporter.otlp.protocol", container.getOtlpProtocol());
-        }
-        if (catalog != null && catalog.hasMicrometerOtlp()) {
-            // always use http -- as that's what Micrometer supports
-            containerConfigs.put("quarkus.micrometer.export.otlp.url",
-                    String.format("http://%s:%s/v1/metrics", host,
-                            container.getMappedPort(ContainerConstants.OTEL_HTTP_EXPORTER_PORT)));
+        containerConfigs.putAll(config(ContainerConstants.GRAFANA_PORT, host));
+        containerConfigs.putAll(config(ContainerConstants.OTEL_HTTP_EXPORTER_PORT, host));
+        // Iff GRPC is the hotel protocol, overwrite the otel-collector.url we just wrote with the correct grpc one, and set up the otlp endpoints
```

Comment:
Iff, hotel? :-)

Reply:
Argh! Even the second time round I didn't see the problem. 🤦‍♀️

I thought you were saying "iff is a hotel chain" and I googled and found a hotel chain called IFF. But no, you were asking "why is the word 'hotel' randomly in this comment?"

And the answer is "I have no idea." 😳

Reply:
I guess my computer helpfully auto-corrected OTLP to hotel ... fixed and squashed.